src/transformers/models/whisper/convert_openai_whisper_to_tfms.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
```suggestion
"""Convert Whisper checkpoints from the original repository. URL: https://github.com/openai/whisper"""

```

Reply:
Also not sure what 'tfms' means, in the name of the conversion script

Reply:
TransForMerS, I copied that from speech_to_text, we can just rename it if needed

Reply:
That's a very weird abbreviation, feel free to rename